extensions/wikia/MultiTasks/SpecialMultiWikiFinder_body.php

```diff
@@ -13,14 +13,10 @@ function __construct() {
 	function execute($page = null, $limit = "", $offset = "", $show = true) {
 		global $wgRequest, $wgUser, $wgOut;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgUser" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgOut" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)